### ISSUE TITLE
Callability on windows?

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/configurable-http-proxy-feedstock#50]
### Comment:

I've never actually tried to run any hub stuff on windows, but on this [PR](https://github.com/conda-forge/jupyterhub-feedstock/pull/66), seeing [this error](
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=597858&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f&l=5726):

```bat
(%PREFIX%) %SRC_DIR%>configurable-http-proxy --help 
'configurable-http-proxy' is not recognized as an internal or external command,
operable program or batch file.
```

Perhaps it needs a wrapper `%SCRIPTS%\configurable-http-proxy.cmd` would be more readily picked up by a "dumb" shell?

**Comment by bollwyvl** [comment]
(i've just skipped the check for now)

**Comment by minrk** [comment]
Yeah, I've no idea! I don't really know why anyone would want/need this on Windows, so I'll call it 'unsupported' until someone submits a patch to add the feature.

**Comment by dhirschfeld** [comment]
Once upon a time I had JupyterHub up & running in Windows Containers on a Windows HPC cluster. I think I may have been the only person doing that! 😅 

The code base still contains some Windows compatibility cruft:

![image](https://user-images.githubusercontent.com/881019/216473370-ecebac33-9478-4626-b23b-66672f01cb2e.png)


I've long since moved to Linux k8s as my deployment target so I think it's probably safe to close this as `wontfix` until someone cares enough to fix it.
